x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -374,7 +374,7 @@ export const ActionTypeForm = ({
       throttleUnit={actionThrottleUnit}
       hasAlertsMappings={hasAlertsMappings}
       onNotifyWhenChange={useCallback(
-        (notifyWhen: any) => {
+        (notifyWhen: RuleNotifyWhenType) => {
```

Comment:
nit: The parameter type is already inferred from the prop type:

<img width="636" alt="image" src="https://github.com/elastic/kibana/assets/18363145/15a6f6f1-8889-4245-8b74-b7d5fc9fd396">